src/app/modules/main/chat_section/chat_content/messages/io_interface.nim

```diff
@@ -159,6 +156,9 @@ method resetAndScrollToNewMessagesMarker*(self: AccessInterface) {.base.} =
 method markAllMessagesRead*(self: AccessInterface) {.base.} =
   raise newException(ValueError, "No implementation available")
 
+method requestMoreMessages*(self: AccessInterface) {.base.} =
+  raise newException(ValueError, "No implementation available")
+
```

Comment:
This change seems not needed, but whatever

Reply:
I think it might be needed, it's very upset if I a remove it, complains about AccessInterface something something :)